src/services/form.js

```
@@ -202,77 +218,50 @@ module.exports = class FormsHelper {
-					for (const formVersion of formsVersionData) {
-						if (formVersion.type) {
-							// Create a promise to fetch and cache the complete form
-							const cachePromise = (async () => {
-								try {
-									// Fetch complete form data by type (this should include sub_type)
-									const completeFormData = await formQueries.findFormsByFilter(
-										{ type: formVersion.type },
-										[tenantCode]
-									)
+				formsVersionData = await cacheHelper.formVersions.get(tenantCode, orgCode, formsVersionData)
+			} catch (Error) {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)